src/Octoshift/AzureApi.cs

```diff
@@ -57,6 +58,26 @@ public virtual async Task<Uri> UploadToBlob(string fileName, byte[] content)
             return GetServiceSasUriForBlob(blobClient);
         }
 
+        public virtual async Task<Uri> UploadToBlob(string fileName, Stream content)
+        {
+            var containerClient = await CreateBlobContainerAsync();
+            var blobClient = containerClient.GetBlobClient(fileName);
+
+            var options = new BlobUploadOptions
+            {
+                TransferOptions = new Azure.Storage.StorageTransferOptions()
+                {
+                    InitialTransferSize = 4 * 1024 * 1024,
+                    MaximumTransferSize = 4 * 1024 * 1024
```

Comment:
these magic numbers should be constants.

Reply:
Fixed in 0818fd2d851be343966f10bad0afcf0f97738b72